src/components/Home.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import Coinsdisplay from './coins';
+import { displayCoin } from '../redux/home/home';
+import './Home.css';
+
+const Coin = () => {
+  const allcoin = useSelector((state) => state.covid);
```

Comment:
* Your project lacks the filter using given parameters functionality, for example 

> When the page loads, the web app shows the list of categories that could be filtered out by some parameter, let's say by the category name.
* Kindly fix this 🙏

Reply:
I made the filter on different coins.

Reply:
It is based on the key imported as v4uuid from uuid.
So I did not make any other filter.